Moviebuff/Models/MovieViewModel.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+
+class MovieViewModel: ObservableObject{
+    private let apiKey = "api_key=e418fdbca3c905f8b53cababd0a32c2f"
```

Comment:
It's always a good idea to keep API keys and secret keys out of git. There are some good ways to do this but usually you have a secrets file in on your machine that is ignored in `.gitignore` but the app can read it it.

Also this is duplicated below on line 35.